package.json

```diff
@@ -62,11 +62,13 @@
     "asyncbox": "2.x",
     "axios": "^0.x",
     "bluebird": "3.x",
+    "chalk": "4.1.2",
     "continuation-local-storage": "3.x",
     "find-root": "^1.1.0",
     "lodash": "^4.17.11",
     "longjohn": "^0.2.12",
     "npmlog": "4.x",
+    "rainbow-colors-array": "^0.0.31",
```

Comment:
i would probably not pull in this dep. we could get away with a short list of colours in our source code. besides, this doesn't seem to be an open source project, or at least i couldn't find a github link

Reply:
Yeah, I thought we might be able to pull this out. I can actually just generate it and copy it as a JSON array.

Reply:
@jlipps Fixed!